configs/coins/nuls.json

```diff
@@ -0,0 +1,67 @@
+{
+  "coin": {
+    "name": "NULS",
+    "shortcut": "NULS",
+    "label": "NULS",
+    "alias": "nuls"
+  },
+  "ports": {
+    "backend_rpc": 6001,
+    "backend_message_queue": 38332,
```

Comment:
Please change also the `backend_rpc` and `backend_message_queue` ports to 8053 and 38353 if possible. It seems to me from the code that NULLS does not use mempool but change the `backend_message_queue` anyway so that it matches other coins.